geowebcache/core/src/main/java/org/geowebcache/grid/GridSetBroker.java

```diff
@@ -183,10 +183,24 @@ public DefaultGridsets getDefaults() {
     }
     
     public GridSet getWorldEpsg4326() {
+        // Use the highest priority EPSG:4326 gridset we have (either an overriden one, or the defaults)
+        Optional<GridSet> gridSet = getGridSet("EPSG:4326");
```

Comment:
If you look in DefaultGridsets, you'll see that the name of the default EPSG:4326 gridset is not always "EPSG:4326": https://github.com/GeoWebCache/geowebcache/blob/master/geowebcache/core/src/main/java/org/geowebcache/config/DefaultGridsets.java#L58-L69

As I noted in my initial review, you'll need to get the default names from the DefaultGridsets object

So something like:
```
getGridSet(getDefaults().worldEpsg4326().getName())
```

Reply:
I see now. One more swing in a bit.